src/napari_imagej/widgets/menu.py

```diff
@@ -227,8 +228,13 @@ def get_chosen_layer(self) -> None:
                 self._add_image(image)
 
     def get_active_layer(self) -> None:
-        # Choose the active DatasetView
-        view = ij().get("net.imagej.display.ImageDisplayService").getActiveDatasetView()
+        # HACK: Sync ImagePlus before transferring
```

Comment:
I love that this is clearly marked. Is there an issue in the imagej-legacy tracker about this? If not, you could make one, then link to the issue, with the comment saying that the code can be removed once imagej/imagej-legacy#xyz is fixed and released.

Reply:
Filed imagej/imagej-legacy#286 and commented the code in e0b3064